src/plugins/irc-events/link.js

```diff
@@ -150,3 +151,13 @@ function fetch(url, cb) {
 			cb(data);
 		}));
 }
+
+// https://github.com/request/request/issues/2120
+// https://github.com/nodejs/node/issues/1693
+// https://github.com/alexeyten/descript/commit/50ee540b30188324198176e445330294922665fc
+function escapeHeader(header) {
```

Comment:
A comment to explain this, in addition to the links, would have be nice, for future ourselves.
To keep in mind when we start adding tests to these.